content/post/kubevault-v2022.12.28/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Introducing KubeVault v2022.12.28
+date: "2022-12-28"
+weight: 25
+authors:
+- Abdullah Al Shaad
+tags:
+- cli
+- hashicorp
+- kubernetes
+- kubevault
+- kubevault-cli
+- redis
+- secret-management
+- security
+- vault
+---
+
+We are very excited to announce the release of KubeVault v2022.06.16 Edition. In the release KubeVault `v2022.12.28` a new `SecretEngine` for `RedisDB` has been added, `KubeVault CLI` has been updated accordingly. We're going to discuss some of them in details below.
+
+- [Install KubeVault](https://kubevault.com/docs/v2022.12.28/setup/)
+
+[KubeVault](https://kubevault.com) is a Kubernetes operator for [HashiCorp Vault](https://www.vaultproject.io/). The Vault is a tool for secrets management, encryption as a service, and privileged access management. The KubeVault operator makes it easy to deploy, maintain and manage Vault servers in Kubernetes. It also supports various secret engines management, policy management in the Kubernetes native way.
+
+In this post, we are going to highlight the major changes. You can find the complete commit by commit changelog [here](https://github.com/kubevault/CHANGELOG/blob/master/releases/v2022.12.28/README.md).
+
+- **Redis SecretEngine**
+
+  Now, `Redis` SecretEngine can be enabled, configured & `RedisRole` can also be created with `KubeVault`.
+  Here's a sample yaml for Redis `SecretEngine` & `RedisRole`:
+
+  ```yaml
+  apiVersion: engine.kubevault.com/v1alpha1
+  kind: SecretEngine
+  metadata:
+    name: redis-secret-engine
+    namespace: demo
+  spec:
+    vaultRef:
+      name: vault
+      namespace: demo
+    redis:
+      databaseRef:
+        name: redis
+        namespace: db
+      pluginName: "redis-database-plugin"
+  ```
+
+
+  ```yaml
+  apiVersion: engine.kubevault.com/v1alpha1
+  kind: RedisRole
+  metadata:
+    name: write-read-role
+    namespace: demo
+  spec:
+    secretEngineRef:
+      name: redis-secret-engine
+    creationStatements:
+      - '["~*", "+@read","+@write"]'
+    defaultTTL: 1h
+    maxTTL: 24h
+  ```
```

Comment:
Also, add the `CLI` command for generating `SecretProviderClass` & the dependencies that we updated.